tests/framework/installer/ceph_installer.go

```diff
@@ -843,7 +843,7 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, useH
 		k8sVersion:      version.String(),
 		CephVersion:     cephVersion,
 		cleanupHost:     cleanupHost,
-		changeHostnames: rookVersion != Version1_2 && k8shelp.VersionAtLeast("v1.13.0"),
+		changeHostnames: k8shelp.VersionAtLeast("v1.13.0"),
```

Comment:
While we're at it, let's update this to something newer. Its purpose is just to improve the test matrix of changing host names on some tests and not others
```suggestion
		changeHostnames: k8shelp.VersionAtLeast("v1.18.0"),
```